### PR TITLE
fixed sign in

### PR DESCRIPTION
**Pull request by luj00**

### CODE REVIEW

File: jordinner/src/pages/restaurants/signin.jsx [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+
+    if (!email.includes('@') || !email.includes('.')) {
+      setSuccessMessage('Please enter a valid email address.');
+      return;
+    }
+
+    if (password.length < 6) {
+      setSuccessMessage('Password must be at least 6 characters long.');
+      return;
+    }
```
Comment (Shwarzar) [inline jordinner/src/pages/restaurants/signin.jsx:27]: According to our schema, the password is 8 character's long, but ultimately, there's no need to give validation errors when signing in. We just need to return an error message stating that there's an issue with either the email or password, without specifying

File: jordinner/src/pages/restaurants/signup.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import React, {useState, useEffect} from 'react'
-import Link from 'next/link'
+import React, { useState, useEffect } from 'react';
```
Comment (Shwarzar) [inline jordinner/src/pages/restaurants/signup.jsx:1]: All changes in sign up are visual and nothing coherent requested other than the comment, keep the code as is as it's not related to sign in

File: jordinner/src/pages/restaurants/signin.jsx [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (response.status === 200) {
+        setSuccessMessage('Sign In Successful!');
+        console.log('Signed in:', { email, password });
+        // Redirect the user to the specific page after successful sign-in
+        router.push('/SingleRestaurant/restaurantId'); // Corrected path
```
Comment (Shwarzar) [inline jordinner/src/pages/restaurants/signin.jsx:43]: path should be /restaurants/[restaurantId]

**Review by Shwarzar — CHANGES_REQUESTED** [review]
Requesting changes

**Review by Shwarzar — APPROVED** [review]

**Comment by luj00** [comment]
changed Requested changes

**Pull request merged**